ui/app/mainui/AppMain.qml

```diff
@@ -2789,6 +2807,16 @@ Item {
         }
     }
 
+    Component {
```

Comment:
Would it make sense to move this code to the `HandlersManager` instead so we don't add more stuff directly to `AppMain` if possible?